### PR TITLE
Fix the grid

### PR DESCRIPTION
**Pull request by ndavies93**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,48 +24,69 @@ <h3>September 10, 2022</h3>
...
-</div>
+      <table class="center">
+        <tbody>
+          <tr>
+            <td>
+              <h4>12:00pm - Ceremony</h4>
+            </td>
+            <td>
+              <p>
+                   <a href="https://goo.gl/maps/W3zcz5DmyxBbGxwx9">Jim Charles Point, Kejimkujik</a>
```
Comment (ewhynot18) [inline index.html:36]: Should this be Jim Charle's ?

File: index.html [hunk 2]
```diff
@@ -112,12 +126,19 @@ <h4>Nearby Hotels and Inns</h4>
...
+    </a>
+  </div>
   </div>
 
-    <h4>Accomidations at Kejimkujik</h4>
-    <p>If you're interested in staying in one of Keji's roofed accomidations on the night of the wedding please indicate this in your RSVP as soon as possible.</p>
-    <p>We have a limited number of roofed accomidations blocked for wedding guests.</p>
-    <p><em>Note that you will need to bring your own bedding/sleeping pads for all Keji roofed accomidations</em>
+    <h4>Accommodations at Kejimkujik</h4>
+    <p>If you are interested in staying in one of Keji's roofed accommodations for the night of the wedding please indicate this in your RSVP as soon as possible.</p>
```
Comment (ewhynot18) [inline index.html:139]:
```suggestion
    <p>If you are interested in staying in one of Keji's roofed accommodations for the night of the wedding please indicate this in your RSVP.</p>
```

File: index.html [hunk 3]
```diff
@@ -160,6 +181,8 @@ <h2>Our Gift Regestry</h2>
     <h2>RSVP</h2>
     <p>
       The favour of your response is requested as soon as you can.
```
Comment (ewhynot18) [inline index.html:183]:
```suggestion
      Respond Soon bruh
```

**Review by ewhynot18 — CHANGES_REQUESTED** [review]
Looks great! I made a few nit-pick suggestions. Just on verbiage, etc.

File: index.html (hunk 1 above)
Comment (ndavies93) [reply index.html:36]: no I spelled it correctly

File: index.html (hunk 3 above)
Comment (ndavies93) [reply index.html:183]: no...

**Review by ewhynot18 — APPROVED** [review]
Looks Great!

**Pull request merged**